Add documentation for the new moehlenhoff_alpha2 integration (#21328)
home-assistant/home-assistant.github.io: source/_integrations/moehlenhoff_alpha2.markdown

---
title: Möhlenhoff Alpha 2
description: Instructions on how to integrate a Möhlenhoff Alpha 2 temperature control system into Home Assistant.
ha_category: Climate
ha_release: 2022.1
ha_iot_class: Local Polling
ha_config_flow: true
ha_codeowners:
  - '@j-a-n'
ha_domain: moehlenhoff_alpha2
ha_platforms:
  - climate
---

The `moehlenhoff_alpha2` platform allows you to control a [Möhlenhoff Alpha 2](https://www.moehlenhoff.de/en/products/room-by-room-control/oem-alpha-2-system) temperature control system.

## Setup

Please make sure the base station is turned on and connected to your local network.
{% include integrations/config_flow.md %}

> Fill in the hostname or the IP address of your base.

Reviewer: ```suggestion
Fill in the hostname or the IP address of your base station.
```

If using a hostname, make sure the name resolution works as expected.

> After pressing `Submit` the integration will connect to your base and setup.

Reviewer: ```suggestion
After pressing `Submit` the integration will connect to your base station.
```

> It is also possible to repeat these steps to setup another base.

Reviewer: ```suggestion
It is also possible to repeat these steps to setup another base station.
```

## Climate

The `moehlenhoff_alpha2` climate platform provides current and target temperature
information, HVAC and preset mode.

A climate entity will be created for each area.

> The name of the entity is taken from the name of the heat area defined in the Alpha 2 base.

Reviewer: ```suggestion
The name of the entity is taken from the name of the heat area defined in the Alpha 2 base station.

---
title: Möhlenhoff Alpha 2
description: Instructions on how to integrate a Möhlenhoff Alpha 2 temperature control system into Home Assistant.
ha_category: Climate
ha_release: 2022.1
ha_iot_class: Local Polling
ha_config_flow: true
ha_codeowners:
  - '@j-a-n'
ha_domain: moehlenhoff_alpha2
ha_platforms:
  - climate
---

The `moehlenhoff_alpha2` platform allows you to control a [Möhlenhoff Alpha 2](https://www.moehlenhoff.de/en/products/room-by-room-control/oem-alpha-2-system) temperature control system.

## Setup

> Please make sure the base is turned on and connected to your local network.

Reviewer: Would `base station` instead of `base` be better?

Author: Yes, could be.

Author: Changed it.

> Go to the integrations page in your configuration and click on new integration -> Möhlenhoff Alpha 2.

Reviewer: Below adds a button that the user can click on to automatically open the config flow on their Home Assistant instance.
```suggestion
{% include integrations/config_flow.md %}

Author: Nice, thank you.

Fill in the hostname or the IP address of your base.
If using a hostname, make sure the name resolution works as expected.

After pressing `Submit` the integration will connect to your base and setup.

> No further configuration is required.

Reviewer: Not really necessary

Author: okay

It is also possible to repeat these steps to setup another base.

## Climate

The `moehlenhoff_alpha2` climate platform provides current and target temperature
information, HVAC and preset mode.

> A climate entity will be created automatically for every available heat area.

Reviewer: ```suggestion
A climate entity will be created for each area.

The name of the entity is taken from the name of the heat area defined in the Alpha 2 base.

The state is polled from the base every 60 seconds.

> Please note that after a state change due to the integration,

Reviewer: ```suggestion
Please note that after changing the temperature in Home Assistant,

Author: Much better, thank you.
